tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh

```diff
@@ -123,7 +123,7 @@ case "${TENSORFLOW_TARGET}" in
     ;;
 esac
 
-cmake --build . --verbose -j ${BUILD_NUM_JOBS} -t _pywrap_tensorflow_interpreter_wrapper
+cmake --build . --verbose -j $(nproc) -t _pywrap_tensorflow_interpreter_wrapper
```

Comment:
$BUILD_NUM_JOBS allows one to control whether they want to run on all cores, on a single core (for debugging), or only a subset of cores (in case others are needed for other tasks). Fixing this to `nproc` removes this flexibility.

Instead, I think it is better to define `BUILD_NUM_JOBS` with a default value if not passed in